java.source.base/test/unit/src/org/netbeans/api/java/source/TreeUtilitiesTest.java

```diff
@@ -584,4 +584,12 @@ public void run(CompilationController parameter) throws Exception {
             }
         }, true);
     }
+
+    public void testIsTreeEndsWithComma() throws Exception {
```

Comment:
This function is checking if we are at the end of the compound var declaration. Can you please rename the method accordingly.